src/data_handler.py

```diff
@@ -0,0 +1,160 @@
+
+    def getName():
+        return name
+
+    def _rawToRealValue(rawValue):
+        pass
+
+    def getSensorID():
+        return sensorID
+
+    def getBoardID():
+        return boardID
+
+class PressureSensor(Sensor):
+
+    def _rawToRealValue(rawValue):          #take in as hex value.
+        data = int(rawValue, 16)
+        return data * (1500 / 4095)         #Maps from (0x000,0xFFF) to (0,1500) psi
+
+class TempSensor(Sensor):
+
+    def _rawToRealValue(rawValue):
+        pass #bin dec conversion
+
+class ForceSensor(Sensor):
+
+    def _rawToRealValue(rawValue):
+        pass
+
+
+
+class Valve:
+
+    def __init__(self, valveID, boardID, closedAngle, openAngle, name):
+        self._valveID = valveID
+        self.boardID = boardID
+        self._closeAngle = closeAngle
+        self._openAngle = openAngle
+        self._lastValue = 0.0
+        self.name = name
+
+    def _rawToRealValue(rawValue):
+        pass #figure out conversion
+
+    def getValue():
+        return lastValue
+
+    def open():
+        BoardInterface.writeValve(boardID, valveID, openAngle)
+
+    def close():
+        BoardInterface.writeValve(boardID, valveID, closeAngle)
+
+    def getName():
+        return name
+
+    def getBoardID():
+        return boardID
+
+    def getValveID():
+        return _valveID
+
+
+class SensorManager:
+
+    def __init__(self):
+
+        _sensors = {}
+        _sensorsByBoard = {1 : [], 2 : [], 3 : [], 4 : [], 5 : [] }
+        _sensorsByID = {}
+
+
+        addSensor(SensorEnum.PRESSURE_1, PressureSensor( 1, 1, "Pressure Sensor 1"))
+        addSensor(SensorEnum.TEMPERATURE_1, TemperatureSensor( 1, 2, "Temperature Sensor 1"))
+        addSensor(SensorEnum.FORCE_1, ForceSensor( 1, 3, "Force Sensor 1"))
+
+        """
+        _sensors[SensorEnum.PRESSURE_1] = PressureSensor(1, 1, "Pressure Sensor 1")
+        _sensorsByBoard[1].append( _sensors[SensorEnum.PRESSURE_1] )
+        _sensorsByID[ (1, 1) ] = _sensors[SensorEnum.PRESSURE_1]
+
+        _sensors[SensorEnum.TEMPERATURE_1] = TemperatureSensor(1, 2, "Temperature Sensor 1")
+        _sensorsByBoard[2].append( _sensors[SensorEnum.TEMPERATURE_1] )
+        _sensorsByID[ (1, 2) ] = _sensors[SensorEnum.TEMPERATURE_1]
+
+        _sensors[SensorEnum.FORCE_1] = ForceSensor(1, 3, "Force Sensor 1")
+        _sensorsByBoard[3].append( _sensors[SensorEnum.FORCE_1] )
+        _sensorsByID[ (1, 3) ] = _sensors[SensorEnum.FORCE_1]
+
+        """
+
+        def addSensor(sEnum, sObject):
+            _sensors[sEnum] = sObject
+            _sensorsByBoard[sObject.getBoardID()].append(sObject)
+            _sensorsByID[ (sObject.getSensorID(), sObject.getBoardID()) ] = sObject
+
+
+    class SensorEnum(Enum):
+        PRESSURE_1 = 1
+
+        TEMPERATURE_1 = 2
+
+        FORCE_1 = 3
+
+
+    def getSensor( sensorType ):
+        return _sensors[sensorType]
+
+    def getSensorByID(sensorID, boardID):
+        return _sensorsByID[(sensorID, boardID)]
+
+    def getAllSensors():
```

Comment:
I believe theres a method for getting all of the keys or values out of a dictionary - its <dict>.keys() and <dict>.values(). I would use that instead here.